### PR TITLE
WebUI : WebUI page to list Liftetime Model exceptions : Closes #1367

### LINKED ISSUE #1367: WebUI page to list Liftetime Model exceptions
Motivation
----------
WebUI page to list Liftetime Model exceptions

### PR DESCRIPTION
**Pull request by cserf**
WebUI : WebUI page to list Liftetime Model exceptions : Closes #1367

### CODE REVIEW

File: lib/rucio/web/ui/static/list_lifetime_exceptions.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+   i = Math.floor(Math.log10(bytes) / 3);
+   return parseFloat(bytes / Math.pow(1000, i)).toFixed(2) + ' ' + sizes[i];
+}
+
+$(document).ready(function(){
+    $('#loader').html('<div class="row"><div class="large-1 large-centered columns"><img src="/media/spinner.gif"></div></div>');
+    r.list_lifetime_exceptions({
+         success: function(data) {
+            var dict = {};
+            data = data.split('\n');
```
Comment (tbeerman) [inline lib/rucio/web/ui/static/list_lifetime_exceptions.js]: Not a big thing but maybe a bit cleaner: There is already a `parse_json_stream` function in rucio.js that is used for the other json streaming calls. That will take care of the splitting, checking for empty data and parsing the json for you.

**Comment by cserf** [comment]
Move to parse_json_stream done

**Comment by tbeerman** [comment]
Looks good so far. Just once small thing: there no link anywhere in the top menu for this. Is it supposed to be hidden?

**Comment by cserf** [comment]
About the link, it can be added later

**Review by bari12 — APPROVED** [review]

**Pull request merged**